src/browser.ts

```diff
@@ -61,6 +61,19 @@ export class GrpcClient {
   auth: GoogleAuth;
   promise?: PromiseConstructor;
 
+  /**
+   * Browser version of GrpcClient
+   * A class which keeps the context of gRPC and auth for the gRPC.
+   *
+   * @param {Object=} options.auth - An instance of google-auth-library.
+   * When specified, this auth instance will be used instead of creating
```

Comment:
In our use case, `auth` is required and should be listed as required (see the error message on line 79 below).